py/surveyqa/nightly.py

```diff
@@ -544,6 +584,22 @@ def makeplots(night, exposures, tiles, outdir):
             </ul>
         """.format(**navigation_links)
 
+# TODO: this is the code that updates the hrefs
+        
+    #- Update the navigation hrefs using the cached valus in linking.js
+    template += """
+    <script>
+        function get_dict(dict) {
```

Comment:
Please use a more descriptive name.   get dict for what?  Input dict is what?  It looks like this function is setting navigation elements, not getting a dictionary.  Please add comments in script about what is does, what the input should be, etc., similar to the type of information we include with python docstrings.